### PR TITLE
Added Infinite scroll in native js

### PR DESCRIPTION
**Pull request by srijanB7**
# Added the code for Infinite Scrolling in native js.
Title : Added the code for Infinite scrolling in native js.

Issue No. : Not an issue

Code Stack : HTML, CSS, JS

# Checklist:

☑️ I have solved the infinite-scrolling problem in native JS.
☑️ I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: native/machine-coding/infinite-scroll/index.html [hunk 1]
```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="icon" type="image/svg+xml" href="../../logo.svg" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <script src="index.js" type="module"></script>
+    <script src="../../helpers/navbar.js" type="module"></script>
+    <link rel="stylesheet" href="style.css" />
```
Comment (sadanandpai) [inline native/machine-coding/infinite-scroll/index.html:10]: import the header as done in [counter](https://github.com/sadanandpai/frontend-mini-challenges/blob/main/native/machine-coding/counter/index.html)

Also, remove the other tags

File: native/machine-coding/infinite-scroll/style.css [hunk 2]
```diff
@@ -0,0 +1,15 @@
+@import url(../../helpers/core.css);
```
Comment (sadanandpai) [inline native/machine-coding/infinite-scroll/style.css:1]: can get rid of this as header will have it as per the new changes

File: native/helpers/challenges.js [hunk 3]
```diff
@@ -409,5 +409,10 @@ export const challenges = [
   },
   { title: 'Emoji Editor', link: '#' },
   { title: 'Sort Table', link: '#' },
-  { title: 'Infinite Scroll', link: '#' },
+  { title: 'Infinite Scroll', 
+    link: 'infinite-scroll', 
+    difficulty: 'hard',
```
Comment (sadanandpai) [inline native/helpers/challenges.js:414]: can be moved to medium

**Review by sadanandpai — CHANGES_REQUESTED** [review]

**Comment by srijanB7** [comment]
Made the assigned changes.

**Comment by srijanB7** [comment]
@sadanandpai I have made the changes.

**Comment by sadanandpai** [comment]
@srijanB7 All comments are not yet addressed. 

Have you run the app in local once?

**Comment by srijanB7** [comment]
@sadanandpai I did run it locally. While debugging in one of the commits I left the core css import by mistake. Turns out I was importing the wrong header file( navbar.js instead of header.js  🥲 ). 

Now I have fixed it. Sorry for the inconvenience.

**Review by sadanandpai — APPROVED** [review]

**Pull request merged**